src/service/poc_lora.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+message poc_lora_beacon_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;  
+  bytes entropy = 3;
+  bytes data = 4;
+  uint64 timestamp = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  string datarate = 10;
+  bytes addr_hash = 11;
+  int32 tx_power = 12;
+  bytes signature = 13;
+}
+
+message poc_lora_beacon_resp_v1 { string id = 1; }
+
+message poc_lora_witness_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;
+  bytes packet_hash = 3;
+  uint64 timestamp = 4;
```

Comment:
the SX1302 and SX1303 have a higher resolution packet timer I'd like to collect the value from, if available. Either add a field to track the resolution of this value, or add a secondary field for the more precise time bits that are currently getting truncated.

Reply:
added ts_res field to track resolution